mux-player

```diff
@@ -0,0 +1,97 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+
+  <title>Phil's Players - mux-player</title>
+
+  <!-- Bootstrap -->
+  <link href="css/bootstrap.flatly.min.css" title="default" rel="stylesheet">
+  <link href="css/bootstrap.darkly.min.css" title="dark" rel="stylesheet">
+  <link href="css/ie10-viewport-bug-workaround.css" rel="stylesheet">
+  <link href="css/jumbotron-narrow.css" rel="stylesheet">
+
+  <!-- Style Switcher -->
+  <script src="js/docs-themes.js" type="text/javascript"></script>
+  <script type="text/javascript">set_style();</script>
+
+  <!-- mux-player -->
+  <script src="https://cdn.jsdelivr.net/npm/@mux/mux-player"></script>
+
+  <!-- JQuery -->
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.3/jquery.min.js"></script>
+
+</head>
+
+<body>
+
+  <div class="container">
+    <div class="header clearfix">
+      <nav>
+        <ul class="nav nav-pills pull-right">
+          <li role="presentation"><a href="index.html">Select a Player</a></li>
+          <li role="presentation"><a href="#" onclick='switch_style("dark"); return false;'>Dark</a></li>
+          <li role="presentation"><a href="#" onclick='switch_style("default"); return false;'>Light</a></li>
+        </ul>
+      </nav>
+      <h3 class="text-muted">Phil's Players</h3>
+    </div>
+
+    <div class="jumbotron">
+      <form id="playform">
+        <div class="form-group">
+          <label for="playback-id">Mux Playback ID:</label>
+          <input type="text" class="form-control" id="playback-id" name="playback-id"><br>
+          <button id="play-mux" type="button" class="btn btn-primary btn-lg">Play Mux!</button><br><br>
+        </div>
+
+        <button type="button" id="mux-demo" class="btn btn-info btn-sm">Load Demo Playback ID</button>
+        <button type="button" id="hls-demo" class="btn btn-info btn-sm">Load Demo HLS Manifest</button>
+      </form>
+    </div>
+
+    <div id="player" class="jumbotron">
+    </div>
+
+    <div class="jumbotron">
+      <p>This player is mux-player<br><a href="https://docs.mux.com/guides/video/mux-player">More
+          details can be found here.</a></p>
+      <p><small>mux-player@latest</small></p>
+    </div>
+
+    <script>
+
+      // Play button
+      $("#play-mux").click(function (event) {
+        $("#player").html('<mux-player id="mux-player" controls playback-id="' + $("#playback-id").val() + '" env-key="1o7ecphf34am3kaqquq06n0t3" autoplay></mux-video>');
+      });
+
+      $("#play-hls").click(function (event) {
+        $("#player").html('<mux-video id="mux-player" controls src="' + $("#manifest").val() + '" env-key="1o7ecphf34am3kaqquq06n0t3" autoplay></mux-video>');
+      });
+
+      $("#hls-demo").click(function (event) {
+        $("#manifest").val("https://stream.mux.com/ZqL00aiFhBGYjtzBsk3N74sKyj4AOQEAH.m3u8");
```

Comment:
Should we just delete this functionality for Mux Player, since it's designed to work with a Playback ID?